src/Microsoft.IdentityModel.Clients.ActiveDirectory/Extensibility/ICustomWebUI.cs

```diff
@@ -0,0 +1,65 @@
+﻿// ------------------------------------------------------------------------------
+// 
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+// 
+// This code is licensed under the MIT License.
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// 
+// ------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Extensibility
+{
+    /// <summary>
+    ///     Interface that an ADAL.NET extender can implement to provide their own Web UI in public client applications
+    ///     to sign-in user and have them consented part of the Authorization code flow.
+    ///     ADAL.NET provides an embedded web view for Windows, but there are other scenarios not yet supported.
+    ///     This extensibility point enables them to provide such UI in a secure way
+    /// </summary>
+    public interface ICustomWebUi
+    {
+        /// <summary>
+        ///     Method called by ADAL.NET to delegate the authentication code Web with with the STS
```

Comment:
Is there a reason for the indentation?
Apparently this perturbates the Reference doc generator:
look at the remarks section in https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client.extensibility.icustomwebui.acquireauthorizationcodeasync?view=azure-dotnet#Microsoft_Identity_Client_Extensibility_ICustomWebUi_AcquireAuthorizationCodeAsync_System_Uri_System_Uri_

Reply:
No, I just copy pasted the comments from MSAL. Will fix here and also on MSAL.

Reply:
This was a remnant of resharper autoformatting in the comments.  as we move to vs2019 and editorconfig this won't get repeated any longer.